plasmapy/diagnostics/tests/test_thomson.py

```diff
@@ -693,7 +693,7 @@ def run_fit(
 
     wavelengths = (wavelengths * u.m).to(u.nm)
 
-    true_params = copy.deepcopy(params)
+    true_params = copy.deepcopy(params)  # noqa: F841
```

Comment:
`true_params` isn't used later, but it could be helpful using the pytest flag to show local variables.